controllers/profiles.js

```diff
@@ -0,0 +1,121 @@
+const fs = require("fs")
+const profiles = require("../data/profiles.json")
+
+const errorResHelper = (errorDescription, res, errCode = 400) => {
+  res.status(errCode).json({ error: errorDescription })
+}
+
+const createProfile = (req, res) => {
+  const now2ISO = new Date().toISOString()
+  const missing = []
+  const newProfile = {
```

Comment:
`idProfile` should be `id`, so that when you search the object you write
```object.id```
instead of
```object.idObject```